Dockerfile

```diff
@@ -3,12 +3,10 @@ FROM python:2.7-slim
 MAINTAINER bhearsum@mozilla.com
```

Comment:
I think this file is pretty straightforward - I'm just stripping out all the test-only stuff. It's a PITA to test it now though, because there's no way to fire up prod-style containers without tweaking docker-compose.yml. Perhaps I should add a companion docker-compose.yml that uses this docker file? In the meantime, you can adjust the "dockerfile" option, remove volume mounts, and the entrypoint from admin to make it work.